Add biased parsing incident, and add support for blockquotes (#81)
badnessdotai/catalog: catalog/chatgpt-female-professor-bias.md

---

> title: ChatGPT struggles to parse sentences where feminine pronouns refer to professors

Reviewer: I think "struggles" is a bit weak here since it outright failed in this task.
```suggestion
title: ChatGPT was unable to correctly interpret sentences where the feminine pronoun referred to a professor
```

companies: ["OpenAI"]
categories: ["Harmful Bias"]
demonstrative: true
modalities: ["Chatbot"]
date: 2023-22-04
models: ["GPT-3.5"]
sources:
  - "https://twitter.com/mmitchell_ai/status/1650110045781393410"
  - "https://twitter.com/ndyjroo/status/1649821809154613248"
---

> When faced with an ambiguous sentence in which a feminine pronoun could refer either to a professor or to a graduate student, ChatGPT consistently failed to recognize that the feminine pronoun could refer to a professor.

Reviewer: ```suggestion
When faced with an ambiguous sentence in which a feminine pronoun could refer either to a professor or to a graduate student, ChatGPT consistently failed to recognize that the feminine pronoun could refer to the professor.
```

For example, a user asked:

> In the sentence "The professor married the graduate student because she was pregnant", who was pregnant?

The model answered:

> The sentence is ambiguous, and it is unclear who was pregnant. It could be interpreted in two ways:
>
> 1. The professor married the graduate student, who was pregnant.
> 2. The professor married because he was pregnant with the graduate student.
> 
> Without further context, it is impossible to know which interpretation is correct.

> Note that in sentence (2), the model uses a masculine pronoun to refer to the pregnant professor.

Reviewer: ```suggestion
Note that in sentence (2), the model used a masculine pronoun to refer to the pregnant professor.
```

The model similarly failed when presented with another sentence, this time with context that strongly suggested that "she" referred to a professor:

> In the sentence "The professor told the graduate student she wasn't working hard enough and therefore hadn't finished reading the thesis chapter", who wasn't working hard enough?

> The model answered:

Reviewer: ```suggestion
The model answered, even starting by explicitly stating its incorrect assumption:
```
